### PR TITLE
Fix not updating the player notification when stopping playback

### PR DESCRIPTION
**Pull request by Abestanis**
The `!running` check caused us not to update the state of the player notification when stopping the player. Fixes the first part of #47.

Notification stop button behavior

### CODE REVIEW

File: test/fakes/fake_just_audio.dart [hunk 1]
```diff
@@ -73,6 +73,14 @@ final icyMetadataMessage = IcyMetadataMessage(
   ),
 );
 
+extension SaveCompleter<T> on Completer<T> {
```
Comment (nt4f04uNd) [inline test/fakes/fake_just_audio.dart:76]: you can remove the name to make it non-public like this `extension on Completer<T>`

File: test/logic/player/music_player.dart [hunk 2]
```diff
@@ -0,0 +1,109 @@
+import 'package:audio_service/audio_service.dart';
```
Comment (nt4f04uNd) [inline test/logic/player/music_player.dart:1]: please rename this file into `music_player_test.dart`

File: test/fakes/fake_just_audio.dart (hunk 1 above)
Comment (Abestanis) [reply test/fakes/fake_just_audio.dart]: Nice, didn't know that was possible.

File: test/logic/player/music_player.dart (hunk 2 above)
Comment (Abestanis) [reply test/logic/player/music_player.dart]: 👍 Maybe rename `test/logic/player/queue.dart` as well for consistency.

**Pull request merged**